README.md

```diff
@@ -3,20 +3,44 @@
 > Log json data to amqp from express routes.
```

Comment:
If we've made this less about express, and more about logging generic data to amqp then there's still a few references to express in the README, etc.